certificate-renewal/Certificate Renewal JSON contract.md

```diff
@@ -133,7 +152,12 @@
 	 - 5-**Disburse funds by check**: Withdraw balance by check, close certificate.
 	 - 6-**Suspend**: Suspend certificate activity - funds retained in share.
 
- - ***transferInOut***: A list of shares eligible for transferring certificate balance into or out of shares and loans as a source of funds for increasing certificate balance 
+ - ***transferIn***: A list of shares eligible as a source for transferring funds into the certificate upon maturity (transfer occurs the day before maturity.) 
```

Comment:
minor nit - to stick with Banno convention/vocabulary, could these be changed to `transferTo` and `transferFrom`?